src/js/menu/acquia_lift.personalize.views.js

```diff
@@ -1436,11 +1119,6 @@
       this.listenTo(this.model, 'change:isActive', this.onEditModeChange);
       this.listenTo(this.campaignCollection, 'change:isActive', this.onCampaignChange);
 
-      this.onPageVariationEditModeProxy = $.proxy(this.onPageVariationEditMode, this);
```

Comment:
I think you want to keep this one around.  The event is still being triggered and we do want to have the menu react by ending the edit mode when it is triggered.